package.json

```
@@ -30,7 +30,7 @@
     "eslint-plugin-format": "^0.1.2",
     "nuxt": "^3.14.1592",
     "pinia": "^2.2.6",
-    "typescript": "~5.7.2",
+    "typescript": "5.6.3",
```

```suggestion
    "typescript": "~5.6.3",
```

I upgraded it by accident